### PR TITLE
Patternlab/dp 31038 add contrast settings utilnav

### PR DESCRIPTION
**Pull request by clairesunstudio**
Changed:
  - project: Patternlab
    component: UtilityNav
    description: Enable adding additional panels and triggers to UtilityNav. Cleanup JS overrides between mobile and desktop (#1856)
    issue: DP-31038
    impact: Minor
  - project: Patternlab
    component: FieldSubmit
    description: Restore the pattern and add form options. (#1856)
    issue: DP-31038
    impact: Minor
Fixed:
  - project: Patternlab
    component: UtilityPanel
    description: Fixed UtilityPanel data and patternlab display. (#1856)
    issue: DP-31038
    impact: Minor
Added:
  - project: Assets,Patternlab
    component: Icon
    description: Add contrast eye icon. (#1856)
    issue: DP-31038
    impact: Minor

Figma Design: https://www.figma.com/file/92U9OqT8sNLyk47TPkL66L/Untitled?type=design&node-id=0-1&mode=design&t=7lRNzfjAFe6U3nhO-0
Openmass Implementation: https://github.com/massgov/openmass/pull/2312

### CODE REVIEW

File: packages/patternlab/styleguide/public/index.html [hunk 1]
```diff
@@ -19,6 +19,7 @@
     <link rel="stylesheet" href="./styleguide/css/styleguide.min.css" media="all" />
     <link rel="stylesheet" href="./styleguide/css/prism-typeahead.min.css" media="all" />
     <link rel="stylesheet" href="./styleguide/css/styleguide.custom.css" media="all" />
+      <noscript>Please ensure Javascript is enabled for purposes of <a href="https://userway.org">website accessibility</a></noscript>
```
Comment (clairesunstudio) [inline packages/patternlab/styleguide/public/index.html]:
```suggestion
```

File: packages/patternlab/styleguide/source/assets/js/modules/utilNav.js [hunk 2]
```diff
@@ -1,33 +1,31 @@
...
+    let $panel;
     if ($(this).closest(".ma__header__hamburger__utility-nav--narrow") !== true) {
       $panel = $(this);
     }
     const height = $panel.height();
+    $panel.css('top', '-' + height + 'px');
     const $closeButton = $panel.find('.js-close-util-nav');
 
-    $panel.css('top', '-' + height + 'px');
+    // $panel.css('top', '-' + height + 'px');
```
Comment (tom-fleming) [inline packages/patternlab/styleguide/source/assets/js/modules/utilNav.js]:
```suggestion
```

File: packages/patternlab/styleguide/source/_meta/_00-head.twig [hunk 3]
```diff
@@ -8,6 +8,8 @@
     <link rel="stylesheet" href="../../assets/css/index-generated.css?{{ cacheBuster }}" media="all" />
 
     <script src="../../assets/js/vendor/modernizr.js"></script>
+    
+    <script src="https://cdn.userway.org/widget.js" data-account="zgT5fBBncI"></script>
```
Comment (tom-fleming) [inline packages/patternlab/styleguide/source/_meta/_00-head.twig]: @clairesunstudio Does this only get loaded for the Patternlab preview pages in Mayflower? This script load wouldn't be carried over to openmass, right?

Comment (tom-fleming) [reply packages/patternlab/styleguide/source/_meta/_00-head.twig]: Not seeing this script in the latest update for mayflower-artifacts, so I think we are good to go.

**Comment by tom-fleming** [comment]
@clairesunstudio I'm seeing a slight shift in the search text here that I think is unexpected. Could you take a look? https://output.circle-artifacts.com/output/job/d7b06e67-8c32-4e5e-80ee-74928bad87b4/artifacts/0/backstop/reports/html/index.html

**Review by tom-fleming — APPROVED** [review]
LGTM

**Pull request merged**